src/Devtools/dev-tool-initializer.ts

```diff
@@ -7,15 +7,15 @@ import { DevToolState } from '../common/types/store-data/idev-tool-state';
 import { InspectHandler } from './inspect-handler';
 
 export class DevToolInitializer {
-    protected _chromeAdapter: DevToolsChromeAdapter;
+    protected _devToolsChromeAdapter: DevToolsChromeAdapter;
 
-    constructor(chromeAdapter: DevToolsChromeAdapter) {
-        this._chromeAdapter = chromeAdapter;
+    constructor(devToolsChromeAdapter: DevToolsChromeAdapter) {
```

Comment:
nitpick: should we use parameter properties here? i.e.
```typescript
constructor(protected readonly devToolsChromeAdapter: DevToolsChromeAdapter) {
`````

Reply:
Agree with this, but will address in separate PR